Parser/pegen/peg_api.c

```diff
@@ -4,25 +4,35 @@
 #include "pegen.h"
 
 mod_ty
-PyPegen_ASTFromString(const char *str, PyArena *arena)
+PyPegen_ASTFromString(const char *str, PyObject *filename_ob, PyArena *arena)
 {
-    return run_parser_from_string(str, parse_start, RAW_AST_OBJECT, arena);
+    return run_parser_from_string(str, parse_start, filename_ob, arena);
 }
 
 mod_ty
-PyPegen_ASTFromFile(const char *filename, PyArena *arena)
+PyPegen_ASTFromFile(const char *filename, PyObject *filename_ob, PyArena *arena)
 {
-    return run_parser_from_file(filename, parse_start, RAW_AST_OBJECT, arena);
+    return run_parser_from_file(filename, parse_start, filename_ob, arena);
 }
 
 PyCodeObject *
-PyPegen_CodeObjectFromString(const char *str, PyArena *arena)
+PyPegen_CodeObjectFromString(const char *str, PyObject *filename_ob, PyArena *arena)
 {
-    return run_parser_from_string(str, parse_start, CODE_OBJECT, arena);
+    mod_ty res = run_parser_from_string(str, parse_start, filename_ob, arena);
```

Comment:
Shouldn't this call `PyPegen_ASTFromString()`? That would make the relationship between the two functions clearer. The compiler can auto-inline it if it thinks it'll matter. (And ditto for the next function.)